### PR TITLE
Fix workflow for chaining extension

### PR DESCRIPTION
**Pull request by talvasconcelos**
Splitpayments should be a pass through and when making the splits, it should maintain the original extra "tag"!

Added an amount (probably could use a better naming) to fix the issue of when splitting an incoming payment, it wouldn't take into account the tip on tpos or the producer fee in livestream!

### CODE REVIEW

**Comment by dni** [comment]
LGTM, one more formatting

**Comment by callebtc** [comment]
@talvasconcelos could you `make format` and resolve the conflicts?

**Comment by talvasconcelos** [comment]
Done, @callebtc

**Comment by talvasconcelos** [comment]
Conflicts resolved... again!! @callebtc

**Comment by talvasconcelos** [comment]
Hope this is fine now!! @callebtc

File: lnbits/extensions/livestream/lnurl.py [hunk 1]
```diff
@@ -86,9 +86,13 @@ async def lnurl_callback(
...
     ls = await get_livestream_by_track(track_id)
 
+    extra_amount = amount_received - int(
+        amount_received * (100 - ls.fee_pct) / 100
+    )  # msats
+
     payment_hash, payment_request = await create_invoice(
         wallet_id=ls.wallet,
-        amount=int(amount_received / 1000),
+        amount=int(amount_received / 1000),  # sats
```
Comment (callebtc) [inline lnbits/extensions/livestream/lnurl.py:95]: Related for this PR?

File: lnbits/extensions/livestream/tasks.py [hunk 2]
```diff
@@ -22,6 +22,9 @@ async def wait_for_paid_invoices():
 
 
 async def on_invoice_paid(payment: Payment) -> None:
+    if not payment.extra:
+        return
```
Comment (callebtc) [inline lnbits/extensions/livestream/tasks.py:26]:
```suggestion
```

not needed since yesterday

File: lnbits/extensions/splitpayments/tasks.py [hunk 3]
```diff
@@ -35,45 +36,41 @@ async def on_invoice_paid(payment: Payment) -> None:
...
 
-    logger.debug(f"checking if tagged for {len(targets)} targets")
-    tagged = False
+    logger.debug(f"performing split payments to {len(targets)} targets")
+
+    amount_to_split = payment.amount
+
+    if payment.extra.get("amount"):
+        amount_to_split = (payment.extra.get("amount") or 0) * 1000
+
```
Comment (callebtc) [inline lnbits/extensions/splitpayments/tasks.py:45]:
```
if amount_to_split == 0:
    return
```
or alternatively return an error?

**Pull request merged**